aperturedb/ParallelLoader.py

```diff
@@ -2,117 +2,23 @@
 import time
```

Comment:
This class seems redundant as *Loader translates to essentially parallel queries .
Is there a use of parallel loader which you foresee ?

We should delete this and derive the loaders from parallelquery.